jazz-build-module/service-configuration-data-loader.groovy

```diff
@@ -344,8 +344,6 @@ def loadServiceConfigurationData() {
 			sh "sed -i -- 's/{conf-region}/${region}/g' ./config/stg-config.json"
 			sh "sed -i -- 's/{conf-region}/${region}/g' ./config/prod-config.json"
 		}
-
-	}
```

Comment:
Why did this get removed?

Reply:
removed the code for getting AWS-region from config-  As per latest discussion it is to come from FunctionARN